api/src/opentrons/commands/module_commands.py

```diff
@@ -150,3 +152,64 @@ def thermocycler_wait_for_lid_temp() -> command_types.ThermocyclerWaitForLidTemp
 def thermocycler_close() -> command_types.ThermocyclerCloseCommand:
     text = "Closing Thermocycler lid"
     return {"name": command_types.THERMOCYCLER_CLOSE, "payload": {"text": text}}
+
+
+def heater_shaker_set_target_temperature(
+    celsius: float,
+) -> command_types.HeaterShakerSetTargetTemperatureCommand:
+    formatted_temp = trunc(celsius)
+    text = f"Setting Target Temperature of Heater-Shaker to {formatted_temp} °C"
```

Comment:
These strings are what get displayed in the run log, right?

If yes, I think we should capitalize them like sentences, only capitalizing the first word and product names like "Heater-Shaker".

```suggestion
    text = f"Setting target temperature of Heater-Shaker to {formatted_temp} °C"
```